addons/edi/models/edi_document.py

```diff
@@ -169,6 +176,18 @@ class EdiDocument(models.Model):
         copy=False,
         tracking=True,
     )
+    processing_state = fields.Selection(
+        [
+            ("waiting", "Waiting"),
+            ("preparing", "Preparing"),
+            ("executing", "Executing")
+        ],
```

Comment:
Would it be good to set default `processing_state` as `waiting`?

Reply:
I dont think so as in generall will be empty on create, it will be waiting only when is created from transfer and allow_process config is not set True on transfer